sdk/search/azure-search-documents/azure/search/documents/indexes/models/_models.py

```diff
@@ -151,6 +122,13 @@ def _from_generated(cls, skillset) -> "SearchIndexerSkillset":
         kwargs["skills"] = custom_skills
         return cls(**kwargs)
 
+    def serialize(self):
```

Comment:
Perhaps keep the same method signature for `serialize` that it was before, then propagate the arguments to the internal serialize call?

```python
def serialize(self, keep_readonly: bool = False, **kwargs: Any) -> JSON:
```